TheModernIlluminati/TheModernIlluminati/Pages/Index.cshtml

```diff
@@ -19,19 +19,19 @@
     </ol>
     <div class="carousel-inner">
         <div class="carousel-item" data-interval="2000">
-            <img src="~/images/7.gif" width=100% height="350">
+            <img src="~/images/7.gif" width=1150 height="350" />
         </div>
         <div class="carousel-item" data-interval="2000">
-            <img src="~/images/1.png" width=100% height="350">
+            <img src="~/images/1.png" width=1150 height="350" />
         </div>
         <div class="carousel-item active text-center" data-interval="2000">
-            <img src="~/images/4.png" width=100% height="350">
+            <img src="~/images/4.png" width=1150 height="350" />
         </div>
         <div class="carousel-item" data-interval="2000">
-            <img src="~/images/6.jpg" width=100% height="350">
+            <img src="~/images/6.jpg" width=1150 height="350" />
```

Comment:
On index.cshtml I removed the warning '100% is not a valid value of attribute width' by replacing 100% with equivalent
page size 1150.

Reply:
See notes above.